junifer/data/atlases.py

```diff
@@ -21,8 +21,8 @@
 from ..utils.logging import logger, raise_error
 
 
-if TYPE_CHECKING:
-    from nibabel import Nifti1Image
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Is the pragma needed?

Reply:
Well, it says that this line is not covered otherwise. We can add the exception for `TYPE_CHECKING` in the configuration of coverage, but the next lines will be counted.

Reply:
Then it's fine, let's keep it.